src/api/forms/FormsAPI.js

```diff
@@ -99,6 +100,7 @@ export default class FormsAPI extends EventEmitter {
      *      @property {HTMLElement} element Parent Element to render a Form
      *      @property {function} onChange a callback function when any changes detected
      *      @property {function} onSave a callback function when form is submitted
+     *      @property {function} onCancel a callback function when form is cancelled
      *      @property {function} onDismiss a callback function when form is dismissed
      */
     showForm(formStructure, {
```

Comment:
This API seems to have got a little lost somewhere along the way (prior to this PR). It was supposed to be a general purpose API for rendering a form into a given HTML element (the `element` argument). A convenience function would then be provided for rendering a given form structure into an overlay, which is what this function is doing. These two things have been conflated into a single function, with two logic paths for the two intended purposes, which we should untangle.

The reason I mention this is that the forms API was never supposed to track any overlay state, which is what it's doing by emitting events specific to the active form overlay. This is why `onchange` is passed in as a callback function instead of the FormsAPI itself being an event emitter. The Forms API should have no concept of an "active" form because it could be tracking multiple forms.

See my comments below on `formPropertiesChanged`, Forms API should not be a singleton EventEmitter, we need to change this behavior.